documentation/CAMARA-OIDC-profile.md

```diff
@@ -0,0 +1,1022 @@
+
+    * **_account_selection_required_**
+
+      The End-User is REQUIRED to select a session at the Authorization Server. The End-User MAY be authenticated at the Authorization Server with different associated accounts, but the End-User did not select a session. This error MAY be returned when the `prompt` parameter value in the Authentication Request is `none`, but the Authentication Request cannot be completed without displaying a user interface to prompt for a session to use.
+
+    * **_consent_required_**
+
+      The Authorization Server requires End-User consent. This error MAY be returned when the `prompt` parameter value in the Authentication Request is `none`, but the Authentication Request cannot be completed without displaying a user interface for End-User consent.
+
+
+* **error_description**
+
+  OPTIONAL. Human-readable ASCII text providing additional information, used to assist the client developer in understanding the error that occurred. Values for the `error_description` parameter MUST NOT include characters outside the set %x20-21 / %x23-5B / %x5D-7E.
+
+
+* **state**
+
+  REQUIRED if the `state` parameter was present in the client authorization request. The exact value received from the client.
+
+
+`error_description` parameter is not a valid field to take decisions on the developer side, as its value may vary depending on the server implementation, language, format or context. Therefore, the developer should base their logic only on the `error` parameter. Moreover, `error_description` message is not intended to be displayed to the user, but only to help the developer debug the issue. If the developer needs to provide a custom error message to the user, they should use their own logic to map the error code to a suitable message, taking into account the user’s language, tone and level of detail.
+
+
+For example, the authorization server redirects the user-agent by sending the following HTTP response (with line wraps within values for display purposes only):
+
+```http
+HTTP/1.1 302 Found
+Location: Location: https://client.example.com/cb?
+    error=unsupported_response_type&
+    error_description=Unsupported%20response_type%20value&
+    state=af0ifjsldkj
+```
+
+
+### Access Token Request
+
+The client makes a request to the token endpoint, using the HTTP POST method, by sending the following parameters using the application/x-www-form-urlencoded format with a character encoding of UTF-8 in the HTTP request entity-body:
+
+* **grant_type**
+
+  REQUIRED. Value MUST be set to `authorization_code`.
+
+
+* **code**
+
+  REQUIRED. The authorization code received from the authorization server.
+
+
+* **redirect_uri**
+
+  REQUIRED, if the `redirect_uri` parameter was included in the authorization request, and their values MUST be identical.
+
+
+The client MUST authenticate with the authorization server as described in the [Client Authentication Section](#client-authentication).
+
```

Comment:
Authentication between Aggregator and Operator API server MUST use private_key_jwt because private_key_jwt avoids shared secrets.
Authentication between Client and Aggregator or between Client and Operator API server SHOULD be private_key_jwt, but MAY be if the Client is not capable of using private_key_jwt.

Reply:
I agree with supporting private_key_jwt, and only private_key_jwt ("SHOULD ... BUT ..." leaves it open to interpretations and interoperability issues).

I think it is important to omit references to aggregators in CAMARA; this profile should be about API Clients and API Servers.